docs/tutorials/code/chisel/rockcraft.yaml

```diff
@@ -1,17 +1,16 @@
-name: chisel-openssl
```

Comment:
where did this content (the openssl part of the tutorial) go?

Reply:
it was deleted. The idea is that the tutorials follow a natural and evolving path. I.e. tutorial 1. shows how to install the `hello` pkg. Then tutorial 2. shows how to get the chiselled version of 1. (instead of a completely different openssl image)